llvm/lib/Target/AArch64/AArch64ExpandPseudoInsts.cpp

```diff
@@ -997,9 +997,9 @@ AArch64ExpandPseudo::expandCondSMToggle(MachineBasicBlock &MBB,
   // streaming-compatible function:
   //
   // OrigBB:
-  //   MSRpstatePseudo 3, 0, %0, 0, <regmask>             <- Conditional SMSTOP
+  //   MSRpstatePseudo 3, 0, %0, IfCallerIsStreaming, <regmask>  <- Cond SMSTOP
   //   bl @normal_callee
-  //   MSRpstatePseudo 3, 1, %0, 0, <regmask>             <- Conditional SMSTART
+  //   MSRpstatePseudo 3, 1, %0, IfCallerIsStreaming, <regmask>  <- Cond SMSTART
```

Comment:
Similar to above, does the `%0` need to be removed from this comment?

Reply:
No, because the toggle is conditional, so it needs the runtime value for `pstate.sm`. The order of the operands was wrong however, so I've fixed that.